lapis/src/main/kotlin/org/genspectrum/lapis/silo/SiloClient.kt

```diff
@@ -80,7 +80,7 @@ open class CachedSiloClient(
 ) {
     private val httpClient = HttpClient.newHttpClient()
 
-    @Cacheable(SILO_QUERY_CACHE_NAME, condition = "#query.action.cacheable && !#query.action.randomize")
+    @Cacheable(SILO_QUERY_CACHE_NAME, condition = "#query.action.cacheable && !(#query.action.randomize ?: false)")
```

Comment:
Just to check that I understand the effect of the `randomize` parameter - this actually affects if the query is cached? Maybe we should add an explanation somewhere as I don't find this name very intuitive I thought it was a way to specify if the output order was random

Reply:
Maybe renaming it to `disableCache` would actually be better

Reply:
Ah ok! Yes that makes sense thanks for the clarification!